.prettierrc.js

```diff
@@ -12,4 +12,22 @@ module.exports = {
   jsxBracketSameLine: false, // default
   arrowParens: 'avoid', // default
   endOfLine: 'lf',
+  plugins: ['@ianvs/prettier-plugin-sort-imports'],
+  importOrder: [
+    '^(react)(.*)$',
+    // Special case for @ebay/nice-modal-react.
+    '^@ebay/nice-modal-react$',
+    '<THIRD_PARTY_MODULES>',
+    '^@(?!ebay/nice-modal-react)(.*)$',
+    '',
+    '^/app/(?!assets)(.*)$',
```

Comment:
```suggestion
    '^/(.*)/(?!assets)(.*)$',
```

covers more than just the app this way